lib/eth/EthRpc.js

```diff
@@ -85,10 +87,16 @@ class EthRPC {
       value: amountStr,
       gasPrice
     };
+
+    if (passphrase) {
+      this.emitter.emit('unlockedForOne');
+    }
+
     const result = await this.web3.eth.personal.sendTransaction(
```

Comment:
We also need to emit `unlocked` when the transaction has either been sent or there is an error